src/index.js

```diff
@@ -0,0 +1,274 @@
+'use babel'
+
+/* @flow */
+
+import * as Path from 'path'
+import * as FS from 'fs'
+import {
```

Comment:
Not really against explicitly listing all of these, but any reason you didn't just do `import * from './helpers'`?